python_modules/dagster/dagster/_core/definitions/asset_check_spec.py

```diff
@@ -28,7 +28,7 @@ class AssetCheckSeverity(Enum):
 
 
 @experimental
-@whitelist_for_serdes(storage_name="AssetCheckHandle")
+@whitelist_for_serdes(old_storage_names={"AssetCheckHandle"})
```

Comment:
by leaving this here, you can still materialize assets with checks (but the check results won't be recorded). We can remove this once 1.5 is released to oss.